servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,44 +1,81 @@
 [[removing-service-mesh]]
 = Removing {ProductName}
 
-The following process removes the {ProductShortName} from an existing installation. Log in as a cluster admin and run these commands from any host with access to the cluster.
+The following process allows you to remove the {ProductShortName} from an existing {product-title} instance. Log in as a cluster admin and run these commands from any host with access to the cluster.
 
 [[remove-control-plane]]
 == Remove the control plane
 
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the operator to begin uninstalling everything it installed.
+====
+
+[NOTE]
+====
+You can used the shortened `smcp` command in place of `servicemeshcontrolplanes`.
+====
+
 . Retrieve the name of the installed custom resource with this command:
 +
-
 ----
 oc get servicemeshcontrolplanes -n istio-system
 ----
 
+. Replace `<name_of_custom_resource>` with the output from the previous command, and run this command to remove the custom resource:
 +
-. Remove the custom resource by running the following command:
+----
+$ oc delete servicemeshcontrolplanes -n istio-system <name_of_custom_resource>
+----
+
+
+[[remove-operators]]
+== Remove the operators
+You must remove the Operators to successfully remove {ProductName}. Once you remove the {ProductName} Operator, you must remove the Jaeger Operator and the Kiali Operator.
```

Comment:
Is Jaeger useful by itself? Kiali likely isn't.

Reply:
Jaeger can be used without Istio.  I don't think Kiali is much use without Istio currently.

Reply:
I added the following note:

"If you use Jaeger outside of {ProductName}, you do not need to uninstall the Jaeger Operator as part of this process."

Reply:
Thinking about it a little bit more, we should probably just say "you should remove the Jaeger Operator and the Kiali Operator" because I'm not 100% sure what configuration changes might be necessary for Jaeger to run on its own without Istio sidecars.

Reply:
I more or less reverted this section to its original form and removed the aforementioned note.